include/stout/flags/v1/positional_argument.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+package stout.v1;
+
+// For defining our own options.
+import "google/protobuf/descriptor.proto";
+
+// Message which represents a positional argument.
+// NOTE: a positional argument is a bare value, and
+// its position in a list of arguments identifies it.
+// For example:
+//     echo 'hello' ' ' 'world'
+// 'hello', ' ', 'world' - are the positional args
+// from the command line.
+//
+// In our implementation of flags we expect that this
+// arguments will be passed without any name.(We added
+// 'repeated' string names and deprecate_names for
+// printing out the help message, obviously other users
+// will not use them).
+// Positional arguments might be followed in a random
+// order, at the end of the command line or even might
+// intermingled with other flags or subcommands. For
+// example the following below is the valid input in
+// the command line:
+//     ./program --top_level_flag1=1 pos_arg1 subcommand 45 --flag2='hello' 5
+// So 'pos_arg1', '45', '5' - are valid positional
+// arguments.
+message Argument {
+  // For printing out help.
+  repeated string names = 1;
+  // For printing out help.
+  repeated string deprecated_names = 2;
+  // For printing out help.
+  string help = 3;
+  // Whether or not the field with 'stout.v1.argument'
+  // extension of the user-defined message is required.
+  bool required = 4;
+  // The position of the positional argument in the
```

Comment:
Can you clarify that this is *global* position (so e.g. in the example above, `45` is in position 1?

The fact that this is global position means that we can't have optional positional arguments, right?

Reply:
Yep, we can't have optional pos args.

Reply:
Yep, I think we should remove it cause we have the global position, so there is no any sense to keep 'required' field.